src/main/java/org/skriptlang/skript/bukkit/loottables/LootTableModule.java

```diff
@@ -0,0 +1,100 @@
+package org.skriptlang.skript.bukkit.loottables;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.bukkitutil.NamespacedUtils;
+import ch.njol.skript.classes.ClassInfo;
+import ch.njol.skript.classes.Parser;
+import ch.njol.skript.command.Commands;
+import ch.njol.skript.lang.ParseContext;
+import ch.njol.skript.registrations.Classes;
+import org.bukkit.Bukkit;
+import org.bukkit.Location;
+import org.bukkit.loot.LootContext;
+import org.bukkit.loot.LootTable;
+import org.jetbrains.annotations.Nullable;
+import org.skriptlang.skript.lang.comparator.Comparator;
+import org.skriptlang.skript.lang.comparator.Comparators;
+import org.skriptlang.skript.lang.comparator.Relation;
+import org.skriptlang.skript.lang.converter.Converters;
+
+import java.io.IOException;
+
+public class LootTableModule {
+
+	public static void load() throws IOException {
+		Skript.getAddonInstance().loadClasses("org.skriptlang.skript.bukkit.loottables", "elements");
+
+		// --- CLASSES --- //
+
+		Classes.registerClass(new ClassInfo<>(LootTable.class, "loottable")
+			.user("loot ?tables?")
+			.name("Loot Table")
+			.description("Loot tables represent what items should be in naturally generated containers, what items should be dropped when killing a mob, or what items can be fished. ")
+			.since("INSERT VERSION")
+			.parser(new Parser<>() {
+				@Override
+				public boolean canParse(ParseContext context) {
+					return false;
+				}
+
+				@Override
+				public @Nullable LootTable parse(String s, ParseContext context) {
+					return null;
+				}
+
+				@Override
+				public String toString(LootTable o, int flags) {
+					return "loot table '" + o.getKey().asString() + '\'';
+				}
```

Comment:
`asString` on namespaced key should be paper only, you can remove this as `asString` is the same as `toString`